src/App.js

```diff
@@ -1,7 +1,16 @@
 import "./App.css";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import Main from "./components/Main";
 
 function App() {
-  return <div className="App">Hello World</div>;
+  return (
+    <div className="App">
+      <Header />
+      <Main />
+      <Footer />
+    </div>
```

Comment:
Nice structured and clean! :) 👍

Reply:
Thanks Debs, I got inspired from your PRs :)